src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -272,7 +274,7 @@ export class StepIndexPattern extends Component<StepIndexPatternProps, StepIndex
 
     this.lastQuery = query;
     this.setState({ query, showingIndexPatternQueryErrors: !!query.length });
-    this.fetchIndices(query);
+    this.fetchIndices();
```

Comment:
wont the change to use the query value from state result in the previous value of the query being fetched? The change you made to debounce this function might be the reason you arent seeing it, but removing the debounce would cause that issue. Was there any reason you decided to use the query value from state instead of the way it was initially setup?